nikola/conf.py.in

```diff
@@ -667,6 +667,13 @@ GITHUB_COMMIT_SOURCE = True
 # If set to False, it will sort by filename instead. Defaults to True
 # GALLERY_SORT_BY_DATE = True
 
+# If set to true, then the user can provide an metadata file which contains
+# the order in which photos in a gallery should be presented. This file is
+# named metadata.yml, is YAML and must contain at least a 'name' field. Other
+# recognised field names are 'order' and 'caption'.
+# Defaults to True
+# GALLERY_SORT_BY_META_FILE = True
```

Comment:
This doesn’t need to be a setting. If user provides a metadata.yml file and it contains `order` fields, try to use them. Otherwise, don’t. I can’t think of a legitimate use case that would have a metadata file with order fields and not want to use it.

Reply:
Agreed